src/crate/client/sqlalchemy/tests/dialect_test.py

```diff
@@ -59,31 +67,56 @@ class Character(self.base):
         self.character = Character
         self.session = Session()
 
-    def test_pks_are_retrieved_depending_on_version_set(self):
+    def test_pks_are_retrieved_depending_on_version_set_old(self):
+        """
+        Test the old pk retrieval.
+        """
```

Comment:
```suggestion
```

Doesn't really add any information that's not already present in the test method name